components/cli/vendor/github.com/docker/docker/pkg/homedir/homedir_unix.go

```diff
@@ -1,18 +1,16 @@
+// +build !windows
```

Comment:
Brought in with this PR:
* https://github.com/moby/moby/pull/34040 Split homedir files by operating system